%YAML:1.0
  leftCameraIntrinsicMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [459.1522, 0.2286, 319.3055, 0, 459.0649, 231.4020, 0, 0, 1.0000]
  rightCameraIntrinsicMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 459.1522, 0.2286, 319.3055, 0,  459.0649,  231.4020, 0, 0, 1.0000]
  leftDistCoeffs: !!opencv-matrix
    rows: 5
    cols: 1
    dt: d
    data: [ 0.0416, -0.0314, 0.0078, -0.0027, 0.0002 ]
  rightDistCoeffs: !!opencv-matrix
    rows: 5
    cols: 1
    dt: d
    data: [ 0.0321, 0.0246, -0.0763, 0.0001, -0.0038 ]
  stereoRotationMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 0.999978975982775, 0.004474075442842, 0.004693638393780, -0.004488103629563, 0.999985480696302, 0.002982502706665, -0.004680226303301, -0.003003505537987, 0.999984537098566 ]
  stereoTransVector: !!opencv-matrix
    rows: 3
    cols: 1
    dt: d
    data: [ -113.6300, -0.7166, -4.6768 ]
  leftRectificationMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 9.7809076856529198e-01, -1.8509162249107997e-02, -2.0735442932378600e-01, 1.8136719702732788e-02, 9.9982868033806538e-01, -3.6972113636044497e-03, 2.0738733771805179e-01, -1.4452085958940509e-04, 9.7825889787312437e-01 ]
  rightRectificationMatrix: !!opencv-matrix
    rows: 3
    cols: 3
    dt: d
    data: [ 9.8038831378073410e-01, -1.4513324611653404e-02, -1.9654037145300646e-01, 1.4866086841259809e-02,  9.9988944254184353e-01, 3.1961756753234204e-04, 1.9651400373560429e-01, -3.2351355579215758e-03, 9.8049568088478833e-01 ]
  leftProjectionMatrix: !!opencv-matrix
    rows: 3
    cols: 4
    dt: d
    data: [ 4.4439979946117080e+02, 0., 4.5062024307250977e+02, 0., 0., 4.4439979946117080e+02, 2.3182076263427734e+02, 0., 0., 0., 1., 0. ]
  rightProjectionMatrix: !!opencv-matrix
    rows: 3
    cols: 4
    dt: d
    data: [ 4.4439979946117080e+02, 0., 4.5062024307250977e+02, -4.5356857528017869e+01, 0., 4.4439979946117080e+02, 2.3182076263427734e+02, 0., 0., 0., 1., 0. ]